core/app/models/spree/shipping_manifest.rb

```diff
@@ -7,7 +7,7 @@ def initialize(inventory_units:)
 
   def for_order(order)
```

Comment:
This method exists because of shipping providers packing things for different order into same cartons.